docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/UserSecurityMaint.xml

```diff
@@ -1979,6 +1979,131 @@
+    allows you to set permissions on sub-folders in a Landing zone. This
+    provides a means to grant and deny users permission to areas within a
+    Landing Zone.</para>
+
+    <sect3 id="GS_LandingZoneFeatureAuth">
+      <title>Landing Zone Feature Authorization</title>
+
+      <para>This lists the HPCC Systems Landing Zone using Feature Level
+      Authorization:</para>
+
+      <para><informaltable>
+          <tgroup cols="2">
+            <tbody>
+              <row>
+                <entry>List/search Dropzone files</entry>
+
+                <entry>FileSprayAccess - SecAccess_Read</entry>
+              </row>
+
+              <row>
+                <entry>Spray a file from a Dropzone</entry>
+
+                <entry>FileSprayAccess - SecAccess_Write</entry>
+              </row>
+
+              <row>
+                <entry>Despray a file to a Dropzone</entry>
+
+                <entry>FileDesprayAccess - SecAccess_Write</entry>
+              </row>
+
+              <row>
+                <entry>Read the content of a Dropzone file</entry>
+
+                <entry>FileIOAccess - SecAccess_Read</entry>
+              </row>
+
+              <row>
+                <entry>Write the content of a Dropzone file</entry>
+
+                <entry>FileIOAccess - SecAccess_Write</entry>
+              </row>
+
+              <row>
+                <entry>Upload a file to a Dropzone using ECLWatch:</entry>
+
+                <entry>FileUploadAccess - SecAccess_Full</entry>
+              </row>
+
+              <row>
+                <entry>Download a file from a Dropzone using ECLWatch</entry>
+
+                <entry>FileSprayAccess - SecAccess_Full</entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>To enable access to a feature, set the permission
+      accordingly.</para>
+
+      <para>This may be sufficient level security in some cases, however,
+      additional restrictions may be needed to secure certain files, from
+      certain users or groups. You can use Landing Zone File Scope security to
+      accomplish this. .</para>
+    </sect3>
+
+    <sect3 id="LZ_FileScopes">
+      <title>Landing Zone File Scopes</title>
+
+      <para>File Scope Level Authorization provides a means to secure access
+      to folders within a Landing Zone.</para>
+
+      <para>An HPCC Administrator can define one or more Landing Zone scopes
+      for each folder in an HPCC Landing Zone.</para>
+
+      <para>Each scope is a file folder of an HPCC Landing Zone. Each Landing
+      Zone scope is one HPCC file scope.</para>
+
+      <para>The landing zone file scopes can be defined using ECLWatch for
+      security enabled systems.</para>
+
+      <para><graphic fileref="../../images/LDAP_0058-1.jpg"
+      vendor="ECLWatchSecurity" />To create a new file scope landing zone
+      folder, go to the Security page of ECL Watch, and click on
```

Comment:
Awkward sentence: To create a new file scope landing zone folder. 
Perhaps: To create a new Landing Zone scope